packages/commonwealth/client/scripts/views/pages/MyCommunityStake/NoTransactionHistory/index.ts

```diff
@@ -0,0 +1 @@
+export { NoTransactionHistory as default } from './NoTransactionHistory';
```

Comment:
if you want to export it as default, why not to do this right away in the NoTransactionHistory file? 
so instead of `export { NoTransactionHistory };` => `export default NoTransactionHistory`